src/applications/disability-benefits/all-claims/tests/pages/secondaryOtherSources.unit.spec.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { DefinitionTester } from '../../../../../platform/testing/unit/schemaform-utils.jsx';
+import { mount } from 'enzyme';
+import formConfig from '../../config/form';
+
+describe('Add secondary other sources of information', () => {
+  const {
+    schema,
+    uiSchema,
+  } = formConfig.chapters.disabilities.pages.secondaryOtherSources0;
+
+  it('should render', () => {
+    const form = mount(
```

Comment:
New linter error with `mount()` . need to have a `form.unmount()` at the very end of each test when using `mount()`